portal/src/pages/style.scss

```diff
@@ -1,4 +1,5 @@
 @use "~@fremtind/jkl-core/jkl";
+@use "~@fremtind/jkl-core/core";
```

Comment:
Det virker på de visuelle regresjonstestene som at vi ikke får med `import` av `.scss` fra `.tsx`-filer (se for eksempel InfoCard). Vil tro rotårsaken til det og at vi manglet Core styles uten dette er den samme.

Reply:
Ah, ja ser det nå. Bare de eksemplene der det lastes inn ekstra stiler som tryner... Men er det ikke litt rart at dette dukker opp nå? Det er vel ikke gjort noen endringer i hvordan eksemplene er bygget opp, og Gatsby-oppsettet er vel også det samme (bortsett fra denne ene linja)?

Reply:
Det viste seg å være `sideEffects: false`. For webpack er CSS og SCSS ubrukt i kode, og ble klippet vekk. Vi fikk på en måte testet at tree-shaking og dead code elim funker da 😄 

https://webpack.js.org/guides/tree-shaking/

fix: [`dac2e2b` (#2919)](https://github.com/fremtind/jokul/pull/2919/commits/dac2e2b5f4d1b31485821bf6ad8ec4c7c2769cca)